SPECS/yaml-cpp/yaml-cpp.spec

```
@@ -1,13 +1,12 @@
 Summary:        A YAML parser and emitter for C++
 Name:           yaml-cpp
-Version:        0.7.0
+Version:        0.8.0
 Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/jbeder/yaml-cpp
-Source0:        https://github.com/jbeder/yaml-cpp/archive/%{name}-%{version}.tar.gz
-Patch0:         yaml-cpp-static.patch
+Source0:        https://github.com/jbeder/yaml-cpp/archive/refs/tags/%{version}.tar.gz
```

The mentioned update to build with a different tarball name:
```suggestion
Source0:        https://github.com/jbeder/yaml-cpp/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
```